README.md

```diff
@@ -1,2 +1,51 @@
-# reviewapps-deploy-status
-A Github Action that tests the deploy status of a Heroku Review App
+# Review Apps deployment status
+A Github Action that tests the deploy status of a Heroku Review App.
+
+## Usage:
+* Include the action in the workflow
+    ```yaml
+    name: Review App Test
+
+    on:
+        pull_request:
+            branches:
+            - master
+
+    jobs:
+    review-app-test:
+
+        runs-on: ubuntu-latest
+        
+        steps:
+        - uses: actions/checkout@v1
+        - name: Run review-app test
+        uses: #TODO: After publishing the action
+        env:
+            GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+            span: 10 # optional in seconds
+            statuses: 200, 302, 403 # optional
+    ```
+
+> Note: Work flow should include `pull_request` event.
+
+* `GITHUB_TOKEN` is required to communicate with Github Deployment API.
+* Input description:  
+    All the inputs are optional. If needed, must be provided from the workflow.
+
+    | Name  | Description  | Default   | 
+    |---|---|---|
+    | span | interval to check the build status  | 10  |
+    | statuses | status_codes to be excluded | 200  |
```

Comment:
List of accepted HTTP status codes. If the response status code is not one of these, merging will be blocked.